tests/python/direct/conftest.py

```diff
@@ -12,6 +12,19 @@
 from python.direct_utils import is_pre_volta, check_captured_python_definition
 
 
+def require_device_mem_size_gb(required_gb=32.0, didx=0):
+    # Get properties for the default CUDA device (GPU 0)
+    device_properties = torch.cuda.get_device_properties(didx)
+    total_memory_bytes = device_properties.total_memory
+    total_memory_gb = total_memory_bytes / (1024**3)
```

Comment:
I'd convert gbs to bytes for precision. Floating point operations are lossy sometimes.

Reply:
done